### PR TITLE
onSubmit functions added to DeleteView

### PR DESCRIPTION
**Pull request by tomy2105**
This change is needed for https://github.com/marmelab/ng-admin/issues/1106

### CODE REVIEW

**Comment by wilfredonoyola** [comment]
Hey,

Any news about this.

Thanks

**Comment by floo51** [comment]
Can you rebase ?

**Comment by tomy2105** [comment]
I've rebased.

File: src/View/DeleteView.js [hunk 1]
```diff
@@ -5,6 +5,98 @@ class DeleteView extends View {
...
+     * The function can be asynchronous, in which case it should return
+     * a Promise.
+     *
+     * @example
+     *
+     *     post.deletionView().onSubmitSuccess(['progression', 'notification', '$state', 'entry', 'entity', function(progression, notification, $state, entry, entity) {
+     *         // stop the progress bar
+     *         progression.done();
+     *         // add a notification
+     *         notification.log(`Element #${entry._identifierValue} successfully edited.`, { addnCls: 'humane-flatty-success' });
```
Comment (floo51) [inline src/View/DeleteView.js]: Can you amend this message ? It's not the edit view anymore. (same 3 lines below)

File: src/View/DeleteView.js [hunk 2]
```diff
@@ -5,6 +5,98 @@ class DeleteView extends View {
...
+     * The function can be asynchronous, in which case it should return
+     * a Promise.
+     *
+     * @example
+     *
+     *     post.deletionView().onSubmitError(['error', 'form', 'progression', 'notification', function(error, form, progression, notification) {
+     *         // stop the progress bar
+     *         progression.done();
+     *         // add a notification
+     *         notification.log(`Some values are invalid, see details in the form`, { addnCls: 'humane-flatty-error' });
```
Comment (floo51) [inline src/View/DeleteView.js]: This example doesn't make sense on the `deletionView`.

**Comment by tomy2105** [comment]
Changed wording a bit to make more sense with delete examples.

**Pull request merged**